<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_main</class>
 <widget class="QWidget" name="Form_main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>620</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1220</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>BBDown - GUI</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>favicon.ico</normaloff>favicon.ico</iconset>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>571</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QCheckBox" name="checkBox_ffmpeg">
      <property name="text">
       <string>ffmpeg</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_ffmpeg">
      <property name="text">
       <string/>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_ffmpeg">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>浏览</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>571</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QCheckBox" name="checkBox_aria2c">
      <property name="text">
       <string>aria2c</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_aria2c">
      <property name="text">
       <string/>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_aria2c">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>浏览</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>571</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_dir">
      <property name="text">
       <string>下载目录</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_dir">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_dir">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>浏览</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>191</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>下载源</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>20</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="radioButton_1">
       <property name="text">
        <string>网页端（无参数）</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_2">
       <property name="text">
        <string>TV端（-tv）</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_3">
       <property name="text">
        <string>APP端（-app）</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_4">
       <property name="text">
        <string>国际端（-intl）</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>20</y>
     <width>241</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>下载视频格式</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>224</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QRadioButton" name="radioButton_5">
       <property name="text">
        <string>优先下载可用编码（无参数）</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_6">
       <property name="text">
        <string>只下载hevc编码（-hevc)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_7">
       <property name="text">
        <string>只下载avc编码(-avc)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_8">
       <property name="text">
        <string>只下载av1编码(-av1)</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>账户</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>111</width>
      <height>100</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QPushButton" name="pushButton_login">
       <property name="text">
        <string>登录</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_logintv">
       <property name="text">
        <string>登录（TV端）</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>571</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>视频地址</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_url">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>24</height>
       </size>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;视频地址 或 av bv BV ep ss&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="whatsThis">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_25">
      <property name="text">
       <string>下载全部分p</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_download">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>360</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>下载</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_advanced">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>360</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>高级选项&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_about">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>关于</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>180</y>
     <width>181</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>mp4box</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_10">
     <item>
      <widget class="QCheckBox" name="checkBox_1">
       <property name="text">
        <string>使用mp4Box来混流</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_23">
       <property name="text">
        <string>设置mp4box的路径</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_6">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;mp4box的路径&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;路径不要包含空格&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>20</y>
     <width>211</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>cookies</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>187</width>
      <height>113</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_13">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QCheckBox" name="checkBox_22">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;设置access_token用以下载TV/APP接口的会员内容&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>单独设置access_token</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_5">
         <property name="placeholderText">
          <string>access_token</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QCheckBox" name="checkBox_21">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;设置字符串cookie用以下载网页接口的会员内容&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>单独设置cookie</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_4">
         <property name="placeholderText">
          <string>cookie</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>20</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>下载选项</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>121</width>
      <height>99</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QCheckBox" name="checkBox_9">
       <property name="text">
        <string>仅下载音频</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_10">
       <property name="text">
        <string>仅下载视频</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_11">
       <property name="text">
        <string>仅下载字幕</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_17">
       <property name="text">
        <string>下载弹幕</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_7">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>180</y>
     <width>221</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>文件名选项</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_14">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>179</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_14">
     <item>
      <widget class="QCheckBox" name="checkBox_18">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;为文件加入清晰度后缀，如XXX[1080P 高码率]&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>文件名包含清晰度后缀</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_12">
       <property name="text">
        <string>不给分P序号补零</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_19">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;多P时，不要加入分P前缀，如[P1],[P2]等&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>不加入分P前缀</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_8">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>310</y>
     <width>191</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>分p</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>174</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_12">
     <item>
      <widget class="QCheckBox" name="checkBox_5">
       <property name="text">
        <string>展示所有分P标题</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QCheckBox" name="checkBox_8">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;选择指定分p或分p范围：(-p 8 或 -p 1,2 或 -p 3-5 或 -p ALL)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>指定下载分p</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_2">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;选择指定分p或分p范围：(-p 8 或 -p 1,2 或 -p 3-5 或 -p ALL)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="placeholderText">
          <string>如1,2或3-5或ALL</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <item>
        <widget class="QCheckBox" name="checkBox_24">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;设置下载合集分P之间的下载间隔时间(单位: 秒, 默认无间隔)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>分P下载时间间隔</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_7">
         <property name="placeholderText">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_9">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>20</y>
     <width>201</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>交互选项</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_15">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>181</width>
      <height>99</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_15">
     <item>
      <widget class="QCheckBox" name="checkBox_4">
       <property name="text">
        <string>交互式选择清晰度</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_2">
       <property name="text">
        <string>仅解析而不进行下载</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_3">
       <property name="text">
        <string>不显示所有音视频流</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_13">
       <property name="text">
        <string>输出调试日志</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_10">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>180</y>
     <width>151</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>跳过选项</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_16">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>131</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_16">
     <item>
      <widget class="QCheckBox" name="checkBox_15">
       <property name="text">
        <string>跳过字幕下载</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_16">
       <property name="text">
        <string>跳过封面下载</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_14">
       <property name="text">
        <string>跳过混流步骤</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_11">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>310</y>
     <width>221</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>其他</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_17">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>198</width>
      <height>139</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_17">
     <item>
      <widget class="QCheckBox" name="checkBox_7">
       <property name="text">
        <string>使用多线程下载</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QCheckBox" name="checkBox_6">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;调用aria2c进行下载时的代理地址配置&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>aria2c的代理地址</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_1">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;调用aria2c进行下载时的代理地址配置&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QCheckBox" name="checkBox_20">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;设置混流的音频语言(代码)，如chi, jpn等&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>设置混流的音频语言代码</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_3">
           <property name="placeholderText">
            <string>如chi,jpn</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>440</y>
     <width>571</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string> BBDown</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_bbdown"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_bbdown">
      <property name="text">
       <string>浏览</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
